src/pug/includes/head.pug

```diff
@@ -7,6 +7,12 @@ meta(http-equiv="X-UA-Compatible" content="IE=edge")
 
 meta(name="description" content="swag opportunities for developers")
 
+meta(name="og:title" content="devSwag")
+meta(name="og:url" content="https://devswag.io/")
+meta(name="og:type" content="website")
+meta(name="og:description" content="swag opportunities for developers")
+meta(name="og:image" content="https://res.cloudinary.com/pul/image/upload/v1632838202/opensource/devswag_thumbnail.png")
```

Comment:
Can you please version this image inside the repository? It could go to  `/assets/img/devswag-card.png`.